package.json

```diff
@@ -161,7 +161,7 @@
     "commander": "^3.0.2",
     "core-js": "^3.6.5",
     "deep-freeze-strict": "^1.1.1",
-    "del": "^5.1.0",
+    "del": "^6.1.1",
```

Comment:
Although this is a major version bump of a direct dependency, it should be safe to backport, because [the only breaking change in v6 is requiring `Node.js` 10](https://github.com/sindresorhus/del/releases/tag/v6.0.0). Our `1.x` branch is already on Node.js 10.

Reply:
Why would that matter - none of them can specify a node engine < 10, right?